include/srtp.h

```diff
@@ -338,6 +338,7 @@ typedef struct srtp_policy_t {
                                 /**< this stream.                        */
     srtp_master_key_t **keys;   /** Array of Master Key structures       */
     size_t num_master_keys;     /** Number of master keys                */
+    bool use_mki;               /** Whether MKI is in use                */
```

Comment:
Per https://www.rfc-editor.org/rfc/rfc3711#section-3.2.1, if MKI is used the length should remain unchanged for the duration of the cryptographic context. Should the `mki_size` be defined here instead of `srtp_master_key_t`?

Reply:
I agree based on the RFC, but that is definitely not what was implemented before. I will change it has it does state "MUST" for the length being the same.